NEWS.md

```diff
@@ -1,3 +1,10 @@
+## 14.1.2 2020-04-09
+
+* Fix processing of job profiles in json format by MatchEngine (MODINV-271)
+* Fix matching on Instance fields (MODINV-275)
+* Enable jackson serialization for EffectiveCallNumberComponent (MODINV-278)
+* Increased memory allocation (in addition to changes in mod-pubsub MODPUBSUB-73)
```

Comment:
Is there a mod-inventory JIRA issue for this change (if it doesn't fit into any of the ones above, please raise a new issue)?

Reply:
no problem, I'll create one

Reply:
Added ticket https://issues.folio.org/browse/MODINV-279

Reply:
Thanks